diff -u -r old/node_modules/express/lib/request.js new/node_modules/express/lib/request.js
--- old/node_modules/express/lib/request.js	2016-02-07 13:04:41.000000000 +0900
+++ new/node_modules/express/lib/request.js	2016-02-07 13:08:58.000000000 +0900
@@ -296,7 +296,7 @@
 
   // Note: X-Forwarded-Proto is normally only ever a
   //       single value, but this is to be safe.
-  proto = this.get('X-Forwarded-Proto') || proto;
+  proto = this.get('X-Forwarded-Proto') || this.get('CloudFront-Forwarded-Proto') || proto;
   return proto.split(/\s*,\s*/)[0];
 });
 
